docs/release-notes/NuGet-6.12.md

```diff
@@ -7,6 +7,8 @@ ms.topic: conceptual
 
 # NuGet 6.12 Release Notes
 
+> **_NOTE:_** In response to developers' feedback to ensure builds continuity when updating to .NET SDK 9, we have reverted the default value of NuGetAuditMode to `direct` in Visual Studio 17.12.3 and .NET 9.0.101.
```

Comment:
```suggestion
> [!NOTE]
> In response to developers' feedback to ensure builds continuity when updating to .NET SDK 9, we have reverted the default value of NuGetAuditMode to `direct` in Visual Studio 17.12.3 and .NET 9.0.101.
```

Please see the docs on learn.microsoft.com's markdown syntax: https://learn.microsoft.com/en-us/contribute/content/markdown-reference#alerts-note-tip-important-caution-warning